etsy.gemspec

```diff
@@ -31,4 +31,5 @@ Gem::Specification.new do |gem|
   # But then when you load active_support, shoulda-context decides
   # to load MiniTest
   gem.add_development_dependency 'minitest', "=4.7.4"
+  gem.add_development_dependency 'test-unit', "~>3.2.5"
```

Comment:
It's been a while since I looked at this, but wouldn't this use either minitest or test-unit, but not both?

Reply:
All I know is that it would not run, said it could not load test/unit. By including the gem, test-unit then it worked. I tried just commenting  out the require, but then it had other loading issues. I wonder if they took out the test/unit compatibility layer at some point.

Reply:
Yes, looks like ruby 2.2 removed the test/unit wrapper around minitest. Since, the original tests are written for test/unit, removing the minitest works fine.

Reply:
Ah, right. Yes that makes sense. Would you go ahead and remove the minitest reference as part of this PR?